runner/app/live/streamer/process_guardian.py

```diff
@@ -3,9 +3,8 @@
 import time
 from typing import Optional
 import abc
-from collections import deque
-from trickle import InputFrame, OutputFrame
 
+from ..trickle import InputFrame, OutputFrame
```

Comment:
I don't Python so good, this is new syntax to me

Reply:
This is making a relative import instead of an absolute one that relies on monkey patching the `syspath` to load the modules under `live` folder as absolute [here](https://github.com/livepeer/ai-worker/blob/05d6d6ff00cfa44938321f4a78ca9f0085b35590/runner/app/live/infer.py#L14-L16).

This is really annoying though because VS Code doesn't recognize the monkey patch so it keeps erroring on those imports. I'll start switching to relative imports wherever I can and at some point we may be able to enable a linter 😋

Reply:
lol I actually had to revert this 🥲 
```
Traceback (most recent call last):
  File "/app/app/live/infer.py", line 12, in <module>
    from streamer import PipelineStreamer, ProcessGuardian
  File "/app/app/live/streamer/__init__.py", line 1, in <module>
    from .streamer import PipelineStreamer
  File "/app/app/live/streamer/streamer.py", line 12, in <module>
    from .process_guardian import ProcessGuardian, StreamerCallbacks
  File "/app/app/live/streamer/process_guardian.py", line 7, in <module>
    from ..trickle import InputFrame, OutputFrame
ImportError: attempted relative import beyond top-level package
```